scripts/scil_concatenate_dwi.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Concatenate DWI, bval and bvecs together. File must be specified in matching
+order. Default data type will be the same as the first input DWI.
+"""
+
+import argparse
+
+from dipy.io import read_bvals_bvecs
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('out_dwi',
+                   help='The name of the output DWI file.')
+    p.add_argument('out_bvals',
```

Comment:
out_bval and out_bvec (just one file)
```suggestion
    p.add_argument('out_bval',
```